wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/document/DocumentInfoScreen.kt

```diff
@@ -51,11 +59,18 @@ import com.android.identity_credential.wallet.ui.KeyValuePairText
 import com.android.identity_credential.wallet.ui.ScreenWithAppBarAndBackButton
 import com.android.identity_credential.wallet.ui.durationFromNowText
 import kotlinx.coroutines.launch
+import java.text.SimpleDateFormat
+import java.util.Date
```

Comment:
Please use the Kotlin versions of this instead.

Reply:
Done.